src/v3/src/components/Link/Link.tsx

```diff
@@ -36,23 +40,31 @@ const Link: UISchemaElementComponent<{
 
     onSubmitHandler({
       params: actionParams,
+      includeData: Boolean(includeData),
       actionFn,
       step,
+      stepToRender,
     });
   };
 
+  const hasAction = actionFn !== undefined || step !== undefined;
+
   return (
-    <Box marginTop={4}>
+    <Box marginTop={2}>
```

Comment:
How does this change affect other views that currently use this component?

Reply:
hmm, interesting, not sure when this one is touched, probably will change it back

Reply:
Actually this is a mistake I meant to change it back to 4